spec/support/features/page_helpers.rb

```diff
@@ -390,6 +390,18 @@ def admin_remove_lead_school_access_page
       @admin_remove_lead_school_access_page ||= PageObjects::SystemAdmin::LeadSchools::RemoveAccessConfirmation.new
     end
 
+    def admin_uploads_page
+      @admin_uploads_index_page || PageObjects::SystemAdmin::Uploads::Index.new
```

Comment:
Typo. Same for the other two below.

```suggestion
      @admin_uploads_index_page ||= PageObjects::SystemAdmin::Uploads::Index.new
```

Reply:
good spot! thanks